README.md

```diff
@@ -114,6 +114,7 @@ The [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDev
 |23| Bot configuration app with type ahead search control.|This sample demonstrates the features of bot configuration and reconfiguration for both teams and group chats, including a type-ahead search (static and dynamic) control on Adaptive Cards.|[View][BotConfigurationApp#csharp]|[View][BotConfigurationApp#nodejs
 |23| Bot configuration app with auth |This sample demonstrates the features of bot configuration auth and reconfiguration for both teams and group chats.|[View][BotConfigurationAppAuth#csharp]|[View][BotConfigurationAppAuth#nodejs]
 |24| Tag mention bot  | This sample app demonstrates the use of tag mention funtionality in teams scope using Bot Framework. |[View][Tagmention#csharp]|[View][Tagmention#nodejs] ![toolkit-icon](assets/toolkit-icon.png) | | |
+|25| Apps in Federated | This sample showcases how apps function within federated group chats. ||[View][bot-feed-members#js] ![toolkit-icon](assets/toolkit-icon.png) 
```

Comment:
Need to change description based on below documnet
https://review.learn.microsoft.com/en-us/microsoftteams/platform/concepts/build-and-test/shared-channels?branch=pr-en-us-10792&tabs=bot

Reply:
Fixed